### PR TITLE
use comments in Configuration as info

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by OskarStark** [comment]
ping @rande

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -29,6 +29,13 @@ public function getConfigTreeBuilder()
         $treeBuilder = new TreeBuilder();
         $rootNode = $treeBuilder->root('sonata_notification')->children();
 
+        $iterationListenersInfo = implode(PHP_EOL, array(
```
Comment (Soullivaneuh) [inline DependencyInjection/Configuration.php]: Why not just use `<<<EOF` method?

File: DependencyInjection/Configuration.php [hunk 2]
```diff
@@ -36,11 +43,23 @@ public function getConfigTreeBuilder()
                 ->children()
                     ->arrayNode('doctrine')
                         ->children()
-                            ->scalarNode('message_manager')->defaultValue('sonata.notification.manager.message.default')->end()
-                            ->scalarNode('max_age')->defaultValue(86400)->end() # max age in second
-                            ->scalarNode('pause')->defaultValue(500000)->end()  # delay in microseconds
-                            ->scalarNode('batch_size')->defaultValue(10)->end() # number of items on each iteration
+                            ->scalarNode('message_manager')
+                                ->defaultValue('sonata.notification.manager.message.default')
```
Comment (Soullivaneuh) [inline DependencyInjection/Configuration.php]: Not related code fine tuning should be on an another PR.

Comment (OskarStark) [inline DependencyInjection/Configuration.php]: :+1:

File: DependencyInjection/Configuration.php (hunk 1 above)
Comment (OskarStark) [inline DependencyInjection/Configuration.php]: will do

**Comment by Soullivaneuh** [comment]
RTM for me. :+1:
